### PR TITLE
Migrate plantronics-hub to caskroom/drivers

### PR DESCRIPTION
**Pull request by commitay**
Updated `tap_migrations.json`

cask has hardware dependencies.

Closes https://github.com/caskroom/homebrew-cask/issues/35433

https://github.com/caskroom/homebrew-drivers/pull/72

### CODE REVIEW

File: tap_migrations.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
   "pgloader": "homebrew/core",
   "phantomjs": "homebrew/core",
   "pl2303": "caskroom/drivers",
+  "plantronics-hub": "caskroom/drivers",
```
Comment (reitermarkus) [inline tap_migrations.json]: This doesn't work for Casks.

Comment (commitay) [inline tap_migrations.json]: > This has no effect for Casks.

I don't understand?

Comment (commitay) [inline tap_migrations.json]: Ping @reitermarkus , @vitorgalvao 

I'm not sure what I should do with this as almost everything in `caskroom/drivers` has an entry in `tap_migrations.json`?

Comment (vitorgalvao) [inline tap_migrations.json]: I’d leave it. It has no effect so it’s not detrimental, and if it ever has an effect, it’s already done.

`conflicts_with` is in a similar position. It exists but does nothing yet.

**Pull request merged**